aries-site/src/data/structures/templates.js

```diff
@@ -337,35 +345,52 @@ export const templates = [
     ],
   },
   {
-    name: 'Persistent Navigation',
+    name: 'Reverse Navigation',
     available: true,
     cardOrder: 100,
-    description: `Navigation choices which persist throughout the application
-    structure, presenting consistent, easy to click-through choices.`,
+    description: `Allows user to move back through previous screens
+    chronologically as you would on a browser hierarchically within
+    an application or website.`,
     preview: {
+      background: 'background-front',
       image: {
         src: {
-          light: '/templateImages/template-preview-persistent.svg',
-          dark: '/templateImages/template-preview-persistent-invert.svg',
+          light: '/templateImages/ReverseNavigation.svg',
+          dark: '/templateImages/ReverseNavigation-invert.svg',
         },
-        alt: 'HPE Peristent Navigation Preview',
+        alt: 'Reverse Navigation Navigation Preview',
+        fit: 'contain',
       },
     },
-    seoDescription: `Persistent navigation pattern. What, why, and when to use 
-    persistent navigation.`,
-    relatedContent: ['Navigation', 'Hub and Spoke Navigation', 'Header'],
-    sections: [
-      'When to Use Persistent Navigation',
-      'Header as Persistent Navigation',
-      'Minimal Sidebar',
-    ],
-    tags: [
-      'navigation',
-      'navigation patterns',
-      'app level navigation',
-      'application navigation',
-      'sidebar',
-    ],
+    seoDescription: `Allows user to move back through previous screens
+    chronologically as you would on a browser hierarchically within
+    an application or website.`,
+    relatedContent: [],
```

Comment:
```suggestion
    relatedContent: [
        'Navigation',
        'Drill Down Navigation',
        'Side to Side Navigation',
        'Matrix Navigation',
],
```

Reply:
I think these suggestions are fair to put in now